project/api/models/civi.py

```diff
@@ -105,8 +105,15 @@ class Civi(models.Model):
     votes_pos = models.IntegerField(default=0)
     votes_vpos = models.IntegerField(default=0)
 
-    def votes(self):
-        from .activity import Activity
+    def __str__(self):
+        return self.title
+
+    def __unicode__(self):
```

Comment:
I believe with Python 3, all strings are unicode by default. So `__unicode__` is deprecated. Please follow the [directions outlined in Django documentation](https://docs.djangoproject.com/en/1.11/topics/python3/#str-and-unicode-methods).